pinot-minion/src/main/java/org/apache/pinot/minion/event/MinionEventObserverFactory.java

```diff
@@ -31,6 +32,11 @@ public interface MinionEventObserverFactory {
    */
   void init(MinionTaskZkMetadataManager zkMetadataManager);
 
+  /**
+   * Initializes the task executor factory.
```

Comment:
Docs for this and the previous method are exactly the same. Let's reason about the new param `MinionTaskObserverStorageManager` as part of this methods docs.